src/js/flows/ingestFiles.js

```diff
@@ -51,16 +51,23 @@ const validateInput = (paths) => ({
 
 const createDir = () => ({
   async do({dataDir}) {
-    await fsExtra.ensureDir(dataDir)
+    dataDir && (await fsExtra.ensureDir(dataDir))
   },
   async undo({dataDir}) {
-    await fsExtra.remove(dataDir)
+    dataDir && (await fsExtra.remove(dataDir))
   }
 })
 
 const createSpace = (client) => ({
   async do(params) {
-    let {name} = await client.spaces.create({data_dir: params.dataDir})
+    let createParams
+    if (params.dataDir) {
+      createParams = {data_dir: params.dataDir}
+    } else {
+      createParams = {name: params.name}
```

Comment:
Is there a reason why we don't send the name in all cases?

Reply:
I opted to send one **or** the other bc zqd only handles for one (https://github.com/brimsec/zq/blob/master/zqd/space/space.go#L108-L123) (giving preference to `name` by handling it first) and I thought it might confuse anyone looking at this code to see that it sends both but only uses `name`.